VMEncryption/main/patch/UbuntuPatching.py

```diff
@@ -57,14 +57,27 @@ def __init__(self, logger, distro_info):
         self.umount_path = '/bin/umount'
         self.touch_path = '/usr/bin/touch'
 
+    def install_cryptsetup(self):
+        cmd = " ".join(['apt-get', 'update'])
+        self.command_executor.Execute(cmd)
+        
+        packages = ['cryptsetup-bin']
+        cmd = " ".join(['apt-get', 'install', '-y'] + packages)
+        self.command_executor.Execute(cmd)
+
     def install_extras(self):
         """
         install the sg_dd because the default dd do not support the sparse write
         """
         cmd = " ".join(['apt-get', 'update'])
         self.command_executor.Execute(cmd)
 
-        packages = ['at', 'cryptsetup-bin', 'lsscsi', 'python-six', 'python-parted', 'procps', 'psmisc', 'gcc', 'libssl-dev', 'libffi-dev', 'python-dev', 'python-pip']
```

Comment:
Formatting looks weird here on GitHub, probably a mixing tabs and spaces issue

Reply:
[79642f1 
](https://github.com/ejarvi/azure-linux-extensions/pull/7/commits/79642f1717948994d06ccd04710cf1e9b1011175)